MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -58,6 +59,7 @@ bool LayeredVolume::createRasterLayers(const MeshLib::Mesh &mesh,
 	this->_minimum_thickness = minimum_thickness;
 	_nodes = MeshLib::copyNodeVector(bottom->getNodes());
 	_elements = MeshLib::copyElementVector(bottom->getElements(), _nodes);
+	_materials.insert(_materials.begin(), _elements.size(), 0); 
```

Comment:
Semantic difference. For someone who does not know that the vector is empty there is a big difference.
Please document and add an assertion that the `_materials` are empty; or use `resize`.